sys/include/evtimer.h

```diff
@@ -137,6 +137,9 @@ static inline uint32_t evtimer_now_msec(void)
 
 /**
  * @brief   Return the current system time in minutes
+ *
+ * @deprecated wrongly placed convenience function, that
+ *             may not work as expected with ztimer (32 bit)
```

Comment:
Can you explain more?

Reply:
deviding 32 bit by 60 000 will only give you about 16 bit of value, since evtimer doesn't need ztimer to be 64 bit, but this one function which is more a convenience function for evtimers base (xtimer at the time now ztimer) i like to deprecate it

this function has one user (gnrc ipv6 nib abr) which can be rewritten to use msec

Reply:
we could also reduce this to 16 bit to have its values in the same (translated) range as the rest of evtimer (this may also save some bits in nib abr

Reply:
If we are deprecating shouldn't its users be removed?